backend/flaskr/app.py

```diff
@@ -42,36 +38,17 @@ def restaurants():
     """Parse the user's restaurant request and get restaurants from Yelp"""
     args = request.json.split('\n')
     user = args[0]
-    food = args[1]
-    price = args[2]
-    dist = args[3]
-    loc = (args[4], args[5])
-
-    print(user, food, price, dist, loc)
-
-    result = yelp.business_search(term=food, location=loc, radius=dist, price=price)
-    if result["businesses"]:
-        for restaurant in result["businesses"]:
-            if not cache.is_cached(user, restaurant["id"]):
-                cache.add_restaurant(user, restaurant["id"])
-                return {"result": {
-                    "id": restaurant["id"],
-                    "Name": restaurant["name"],
-                    "Location": restaurant["location"],
-                    "Distance": round(restaurant["distance"] / 1609.34, ndigits=1),
-                    "Price": restaurant["price"],
-                    "Rating": restaurant["rating"]
-                }}
-
-    # We exhausted all the available restaurants or there were none at all
-    print("Could not find any restaurants with the given parameters!")
-    return {"result": {
-        "id": "N/A",
-        "Name": "No matches found!",
-        "Distance": "I would walk 500",
-        "Price": "$$$$",
-        "Rating": 5
-    }}
+    search_params = {
+        "food": args[1],
+        "price": args[2],
+        "distance": args[3],
+        "location": (args[4], args[5])
+    }
+
+    print(user, search_params)
+    return {
+        "result": recommender.get_restaurant(user, search_params)
+    }
```

Comment:
nit: this can be one line (fits into our 100 char maximum and is only one item).

Reply:
pick: You're right that it *can* be one line... But personally, if there's ever some non-trivial line embedded in a dictionary like this or an if else statement or whatever, I think it's more aesthetically pleasing to have a list of just keys and values on each line. I respect the comment though as it could be condensed.